src/scripts/app.js

```diff
@@ -259,7 +259,7 @@ export default class InteractiveBook extends H5P.EventDispatcher {
         urlFragments: URLTools.extractFragmentsFromURL(this.validateFragments, this.hashWindow),
         chapters: chapters
       };
-      if (this.getAnswerGiven()) {
+      if (!!this.getAnswerGiven() || this.activeChapter > 0) {
```

Comment:
the !! is just for readability / cleaner to understand
need to check activeChapter to prevent having 0 score if no answer were given